.claude/session_context/docs/code_review_phase_d.md

```
@@ -0,0 +1,303 @@
+**Location**: `internal/resource/fetch.go`
+
+The global HTTP client singleton pattern remains despite efforts to remove it:
+
+```go
+var (
+    globalHTTPClient *HTTPClient
+    clientOnce       sync.Once
+)
+```
+
+**Issues**:
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)